doc/source/user_guide/index.rst

```diff
@@ -34,7 +33,7 @@ This section provides a general overview of PyMAPDL and how you use it.
 
 PyMAPDL overview
 ================
-The :func:`launch_mapdl() <ansys.mapdl.core.launch_mapdl>` function
+The :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>` function
```

Comment:
Function or method for the descriptor after the :func: role?

Reply:
Function :)